tests/network/probes.go

```diff
@@ -101,15 +110,35 @@ var _ = SIGDescribe("[ref_id:1182]Probes", func() {
 				tests.WaitAgentConnected(virtClient, vmi)
 			}
 
-			By("Checking that the VMI and the pod will be marked as ready to receive traffic")
-			Eventually(isVMIReady, 60, 1).Should(Equal(true))
-			Expect(tests.PodReady(tests.GetRunningPodByVirtualMachineInstance(vmi, util.NamespaceTestDefault))).To(Equal(corev1.ConditionTrue))
+			checkStatus(true, corev1.ConditionTrue)
+
+			if disableEnableCycle {
+				By("Disabling the guest-agent")
+				_, err := console.SafeExpectBatchWithResponse(vmi, []expect.Batcher{
+					&expect.BSnd{S: "sudo systemctl disable qemu-guest-agent"},
+					&expect.BExp{R: console.ShellSuccess},
+				}, 120)
+				Expect(err).ToNot(HaveOccurred())
+
+				checkStatus(false, corev1.ConditionFalse)
+
+				By("Enabling the guest-agent again")
+				_, err = console.SafeExpectBatchWithResponse(vmi, []expect.Batcher{
+					&expect.BSnd{S: "sudo systemctl enable qemu-guest-agent"},
+					&expect.BExp{R: console.ShellSuccess},
+				}, 15)
+				Expect(err).ToNot(HaveOccurred())
+
+				checkStatus(true, corev1.ConditionTrue)
+			}
 		},
-			table.Entry("[test_id:1202][posneg:positive]with working TCP probe and tcp server on ipv4", tcpProbe, corev1.IPv4Protocol, false),
-			table.Entry("[test_id:1202][posneg:positive]with working TCP probe and tcp server on ipv6", tcpProbe, corev1.IPv6Protocol, false),
-			table.Entry("[test_id:1200][posneg:positive]with working HTTP probe and http server on ipv4", httpProbe, corev1.IPv4Protocol, false),
-			table.Entry("[test_id:1200][posneg:positive]with working HTTP probe and http server on ipv6", httpProbe, corev1.IPv6Protocol, false),
-			table.Entry("[test_id:TODO]with working Exec probe", createExecProbe(period, initialSeconds, timeoutSeconds, "uname", "-a"), blankIPFamily, true),
+			table.Entry("[test_id:1202][posneg:positive]with working TCP probe and tcp server on ipv4", tcpProbe, corev1.IPv4Protocol, false, false),
+			table.Entry("[test_id:1202][posneg:positive]with working TCP probe and tcp server on ipv6", tcpProbe, corev1.IPv6Protocol, false, false),
+			table.Entry("[test_id:1200][posneg:positive]with working HTTP probe and http server on ipv4", httpProbe, corev1.IPv4Protocol, false, false),
+			table.Entry("[test_id:1200][posneg:positive]with working HTTP probe and http server on ipv6", httpProbe, corev1.IPv6Protocol, false, false),
+			table.Entry("[test_id:TODO]with working Exec probe", createExecProbe(period, initialSeconds, timeoutSeconds, "uname", "-a"), blankIPFamily, true, false),
+			table.Entry("[test_id:6739]should succeed with GuestAgentPing", guestAgentPingProbe, blankIPFamily, true, false),
```

Comment:
the table is already called should succeed, so you can remove it from the descriptions or it will read double.